pkg/connector/client.go

```diff
@@ -140,11 +286,11 @@ func (m *MetaClient) Disconnect() {
 
 // GetCapabilities implements bridgev2.NetworkAPI.
 func (m *MetaClient) GetCapabilities(ctx context.Context, portal *bridgev2.Portal) *bridgev2.NetworkRoomCapabilities {
-	panic("unimplemented")
+	return &bridgev2.NetworkRoomCapabilities{}
 }
 
 // GetChatInfo implements bridgev2.NetworkAPI.
-func (m *MetaClient) GetChatInfo(ctx context.Context, portal *bridgev2.Portal) (*bridgev2.PortalInfo, error) {
+func (m *MetaClient) GetChatInfo(ctx context.Context, portal *bridgev2.Portal) (*bridgev2.ChatInfo, error) {
 	panic("unimplemented")
```

Comment:
@tulir 
Still need to figure out what exactly to do about `GetChatInfo` -- in previous commits, I had a version where I stored all thread metadata separately in-memory (doesn't need to be written to DB because we receive it all at startup in the initial table), but that is obviously suboptimal.

I don't think there is a proper way to accomplish it without the `CreateMatrixRoom` hack I have right now, because otherwise there is nowhere to store thread metadata before the room is created. And the hack doesn't work if the user deletes and recreates the portal without restarting.

(This does not apply to `GetUserInfo` and ghost metadata as far as I can tell, because that can just be set at any time, even before the ghost has joined any rooms)

Reply:
Came up with a solution that works, thought it might be a bit wasteful (depends on if you prioritize memory possibly used by un-bridged chats or network requests):
Just re-requesting the entire initial table and manually parsing out the metadata right there.